ts/environment.ts

```diff
@@ -28,12 +29,18 @@ export const check = (
   ...args: Array<Interpolatable>
 ) => emptyEnvironment.check(s, ...args);
 
+export const build = (
+  s: TemplateStringsArray,
+  ...args: Array<Interpolatable>
+) => emptyEnvironment.build(s, ...args);
```

Comment:
I don't think this will conflict with #262, but if that makes it in first we should switch this to `function` and add a jsdoc:

```suggestion
/**
 * Creates a new shell script `Package`, run in the `emptyEnvironment`.
 *
 * Copy any artifacts that you want to keep from the build in `$out`.
 *
 * Example:
 * ```typescript
 * import * as pkgs from "https://garn.io/ts/v0.0.9/nixpkgs.ts";
 * // Creates a package containing a 4096-bit RSA ssh key
 * garn.build`
 *   ${pkgs.openssh}/bin/ssh-keygen -b 4096 -t rsa -N "" -f $out/id_rsa
 * `;
 * ```
 */
${pkgs.openssh}/bin/ssh-keygen -b 4096 -t rsa -N "" -f $out/id_rsa
export function build(
  s: TemplateStringsArray,
  ...args: Array<Interpolatable>
) {
  return emptyEnvironment.build(s, ...args);
}
```

Or maybe if the ssh example is too complex, maybe just soemthing simple like
```
    ${pkgs.cowsay}/bin/cowsay moo > $out/moo
```